pkg/entity-store/store.go

```diff
@@ -299,6 +430,62 @@ func (es *entityStore) Get(organizationID string, name string, entity Entity) er
 	return nil
 }
 
+func doFilterStat(fs FilterStat, entity Entity) (bool, error) {
```

Comment:
So are we not using SQL to do the actual filtering in the case of Postgres?

Reply:
nope, this function is for libkv filter, please see the function MakeListSQL function in postgres.go